data.go

```diff
@@ -93,8 +93,8 @@ func (s *SessionManager) Load(ctx context.Context, token string) (context.Contex
 func (s *SessionManager) Commit(ctx context.Context) (string, time.Time, error) {
 	sd := s.getSessionDataFromContext(ctx)
 
-	sd.mu.Lock()
-	defer sd.mu.Unlock()
+	sd.mu.RLock()
```

Comment:
@pete-woods I'm not sure that we an use RLock in the `Commit` function --- `sd` is written to in this line: https://github.com/alexedwards/scs/blob/9b079983c0f98c750cdeadf3bca2bd5fe60d35f8/data.go#L101.

Reply:
Thanks - will have a look